test/catch/tests.cpp

```
@@ -1,9 +1,14 @@
 #include <stdexcept>
 
 #include <catch2/catch_session.hpp>
+#include <catch2/catch_test_macros.hpp>
 
 #include "core.hpp"
 
+TEST_CASE("core/fixed_vector/FixedVector constructors") {
```

Cppcheck (2.19.0)

[style] The function 'TEST_CASE' is never used.

(unusedFunction)